### PR TITLE
Fixes for non working links in documentation.

### PR DESCRIPTION
**Pull request by JulianFeinauer**
Hi all,
I just fixed some minor issues with links on the documentation that were not working (and annoyed me during reading).

### CODE REVIEW

File: docs/Documentation/UserGuideV0.7.0/2-Concept.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ IoTDB can support LONG types and DATETIME-DISPLAY types when displaying timestam
 
 * Value
 
-The value of a time series is actually the value sent by a sensor to IoTDB. This value can be stored by IoTDB according to the data type. At the same time, users can select the compression mode and the corresponding encoding mode according to the data type of this value. See [Data Type](需要连接到具体的网页链接Chapter2) and [Encoding](需要连接到具体的网页链接Chapter2) of this document for details on data type and corresponding encoding.
+The value of a time series is actually the value sent by a sensor to IoTDB. This value can be stored by IoTDB according to the data type. At the same time, users can select the compression mode and the corresponding encoding mode according to the data type of this value. See [Data Type](#data-type) and [Encoding](需要连接到具体的网页链接Chapter2) of this document for details on data type and corresponding encoding.
```
Comment (jixuan1989) [inline docs/Documentation/UserGuideV0.7.0/2-Concept.md]: [Encoding](需要连接到具体的网页链接Chapter2)  is:  [Encoding](#encoding)

File: docs/Documentation/UserGuideV0.7.0/2-Concept.md [hunk 2]
```diff
@@ -191,7 +191,7 @@ A column of data contains all values belonging to a time series and the timestam
...
+The time series of FLOAT and DOUBLE type can specify (MAX\_POINT\_NUMBER, see [this page](#iotdb-query-statement) for more information on how to specify), which is the number of digits after the decimal point of the floating point number, if the encoding method is **RLE** or **TS\_2DIFF** (Refer to [Create Timeseries Statement](#chapter-5-iotdb-sql-documentation) for more information on how to specify). If MAX\_POINT\_NUMBER is not specified, the system will use **float\_precision** in the configuration file "tsfile-format.properties" for configuration for the configuration method.
```
Comment (jixuan1989) [inline docs/Documentation/UserGuideV0.7.0/2-Concept.md]: <pre>
**RLE** or **TS\_2DIFF** can be modified as  [**RLE**](#encoding)  or [**TS\_2DIFF**](#encoding)
</pre>

**Comment by JulianFeinauer** [comment]
Hey @jixuan1989 of course you are right with both your comments. I pushed the fixes.

**Comment by jixuan1989** [comment]
That's Cool! 
This is a special welcome from IoTDB by Chinese... :P

**Comment by JulianFeinauer** [comment]
👍 Thank you for the welcome @jixuan1989 and I hope to send some more PRs soon :)

**Pull request merged**